ChangeLog

```diff
@@ -15,6 +15,15 @@ The rules for this file:
 
 ------------------------------------------------------------------------------
 
+15/06/2022 IAlibay
```

Comment:
Wrong date… it’s July. 

And the date on 0.6.0 should also be fixed retroactively